Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -42,15 +42,18 @@ extension TodosViewController: UITableViewDataSource {
 
 extension TodosViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        var removedTodo = todos[indexPath.section].remove(at: indexPath.row)
-        removedTodo.completed = !removedTodo.completed
-        todos[1 - indexPath.section].insert(removedTodo, at: todos[1 - indexPath.section].count)
-        tableView.reloadData()
+        let finishIndexPath = IndexPath(row: todos[1 - indexPath.section].count, section: 1 - indexPath.section)
```

Comment:
When `completing` a `Todo`, lets move its cell to the start of the `completed` section, not the end. The UX's much better than the current state, as it's not entirely apparent what happened when the user taps an `active` `Todo` and completes it.